src/components/dragBoxLayer.ts

```diff
@@ -289,7 +289,7 @@ export module Components {
 
     // Sets resizable classes. Overridden by subclasses that only resize in one dimension.
     protected _setResizableClasses(canResize: boolean) {
```

Comment:
Probably best done as a separate refactor. Plus, it's `protected`, so technically we can't change the API...